### ISSUE TITLE
OneSignal.setDefaultNotificationUrl is not a function

### ISSUE THREAD

**Issue by katsar0v** [OneSignal/OneSignal-WordPress-Plugin#171]
(By the way https://goo.gl/ahXeJq leads to 404)

**Description:**
OneSignal.setDefaultNotificationUrl is not a function error in the console

**Steps to Reproduce Issue:**
1. Install WordPress plugin
2. Use custom init

My custom init:
```
<script>
jQuery(document).on('policy_accepted', function() {
  var OneSignal = window.OneSignal || [];
  OneSignal.push(function() {
    OneSignal.push(["init", window._oneSignalInitOptions]);
  });
});
</script>
```

`policy_accepted` happens whenever the user accepts (clicks on a button) website's policy.

**Full error:**
```
OneSignalStubES5.js:47 TypeError: OneSignal.setDefaultNotificationUrl is not a function
    at (index):11
    at Function.e.processItem (ProcessOneSignalPushCalls.js:5)
    at t.push (OneSignalStubES5.js:22)
    at t.playPushes (OneSignalStubES5.js:44)
    at new t (OneSignalStubES5.js:9)
    at Function.e.addOneSignalPageES5SDKStub (OneSignalShimLoader.js:52)
    at Function.e.start (OneSignalShimLoader.js:65)
    at Object.<anonymous> (sdk.js:13)
    at r (bootstrap:19)
    at bootstrap:68
```

![image](https://user-images.githubusercontent.com/7236157/53871601-d8049c00-3ffc-11e9-9bd8-f085a91d6f0f.png)

**Comment by jkasten2** [comment]
@katsar0v Thanks for reporting, I was able to reproduce this error message and we will have a fix for it in the next release.

This error only happens on browser that do NOT support push notifications. I reviewed our code on how there is a catch for this error but it is still logged to the console. After the SDK logs the error it will correctly continue executing the rest of the javascript on your page.

In summary this is an error message and won't have any effect on your site, it will be cleaned up in the future.
